ui/src/app/guards/access-tasks-guard.service.ts

```diff
@@ -25,7 +25,8 @@ export class AccessTasksGuard implements CanActivate, CanActivateChild {
     }
     return this.serverConfigService.getConfig().flatMap((config) => {
       return this.profileStorageService.profile$.flatMap((profile) => {
-        if ((profile.linkedNihUsername === null || profile.linkedNihUsername === '')
+        if ((profile.eraCommonsLinkedNihUsername === null
```

Comment:
Optional nitpick: It would be nice to check the same field as we check in the backend – eraCommonsCompletionTime. That will set up another nice pattern here, where every access module (regardless of what supplementary data it tracks) is checked against the same Timestamp-type field for completion.